### PR TITLE
created new gdcr event for Bonn

### PR DESCRIPTION
**Pull request by nbulitta**

### CODE REVIEW

File: _data/events/bonn.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "title": "Global Day Of Code Retreat at itemis Bonn",
+  "url": "https://www.meetup.com/de-DE/preview/itemis/events/244448511",
```
Comment (rradczewski) [inline _data/events/bonn.json]: This looks like you accidentally used the preview link for the event on Meetup. Do you want to swap it out for the correct one?

**Comment by apandichi** [comment]
Actually, I think this "preview" URL is actually the new Meetup interface
that anyone can try if they use the "preview" prefix.

File: _data/events/bonn.json (hunk 1 above)
Comment (nbulitta) [reply _data/events/bonn.json]: I think it's the right url. I modified the url in the browser and removed "/preview" but meetup forwarded me to the same url (with "/preview") again. Thanks for asking!

**Comment by rradczewski** [comment]
Uh good catch @apandichi, I'll merge it then :) thanks!

**Pull request merged**